frontend/packages/common-frontend/package.json

```
@@ -45,7 +45,7 @@
     "access": "public"
   },
   "dependencies": {
```

Changing to peer dependencies gives the version choice to the user (or Flow framework). It guarantees that we won't have any dupes.
```suggestion
  "peerDependencies": {
```